### ISSUE TITLE
--url not working( [error]  Error: Public key not registered)

### ISSUE THREAD

**Issue by MeijeSibbel** [Storj/core-cli#22]
### Package Versions

Replace the values below using the output from `npm list storj`. Use ```npm list -g storj``` if installed globally.

```
storj-bridge@1.0.1 /home/gordon/Code/storj-bridge
└── storj@4.0.1  -> /home/gordon/Code/storj-core
```

Replace the values below using the output from `node --version`.

```
v4.5.0
```

### Expected Behavior

```
--url works as expected
```

### Actual Behavior

```
C:\Users\Dani Marion>storj --url http://apipool001.storj.dk login
 [Mon Oct 10 2016 22:03:10 GMT+0200 (W. Europe Daylight Time)] [error]  This device is already paired.

C:\Users\Dani Marion>storj --url http://apipool001.storj.dk list-buckets
 [Mon Oct 10 2016 22:03:31 GMT+0200 (W. Europe Daylight Time)] [error]  Error: Public key not registered


C:\Users\Dani Marion>


```

### Steps to Reproduce

use --url

**Comment by MeijeSibbel** [comment]
Related to https://github.com/Storj/core-cli/issues/23 , using workaround suggested by @littleskunk , the --url works again.

**Comment by MeijeSibbel** [comment]
related to https://github.com/Storj/core-cli/issues/23

**Comment by littleskunk** [comment]
Duplicate to #23

As soon as we fix #23 you should be able to login on both bridges and to use all command on both bridges.
